### PR TITLE
C109

### PR DESCRIPTION
**Pull request by SupriyaPamulapati**
Updated the get api to fetch message along with Twitter Handles in JSON format with the help of pojo classes

### CODE REVIEW

File: src/test/com/test/RetrieveTweetsTest_implement.java [hunk 1]
```diff
@@ -92,7 +93,7 @@ public void testCase_fetchTweet_successCase() throws TwitterException {
         when(s3.getText()).thenReturn("Tweet3");
         when(twitter.getHomeTimeline()).thenReturn(responseList);
         List<String> expected = Arrays.asList("Tweet1", "Tweet2", "Tweet3");
-        List<String> actual = twitterImplement.GetTweets();
+        List<Pojo_TwitterResponse> actual = twitterImplement.GetTweets();
```
Comment (anasuya1117) [inline src/test/com/test/RetrieveTweetsTest_implement.java:96]: function name should never start with uppercase (i.e GetTweets)

**Review by anasuya1117 — APPROVED** [review]
Address the comments and merge

File: src/main/java/UTwitter/service/TwitterImplement.java [hunk 2]
```diff
@@ -18,6 +23,8 @@
     ConfigurationBuilder configurationBuilder;
     TwitterFactory twitterFactory;
     Twitter twitter;
+    User user;
+    Pojo_TwitterResponse pojo_twitterResponse;
```
[the remark this answers is not in the document]
Comment (SupriyaPamulapati) [reply src/main/java/UTwitter/service/TwitterImplement.java]: I have Changed the Naming

File: src/main/java/UTwitter/service/TwitterImplement.java [hunk 3]
```diff
@@ -18,6 +23,8 @@
     ConfigurationBuilder configurationBuilder;
     TwitterFactory twitterFactory;
     Twitter twitter;
+    User user;
```
[the remark this answers is not in the document]
Comment (SupriyaPamulapati) [reply src/main/java/UTwitter/service/TwitterImplement.java]: removed the unused objects

File: src/main/java/UTwitter/service/TwitterImplement.java [hunk 4]
```diff
@@ -34,26 +41,37 @@ public TwitterImplement(TwitterFactory twitterFactory) {
...
     }
 
-    public ArrayList<String> GetTweets() {
-        ArrayList<String> arrayList = new ArrayList<>();
+    public ArrayList<Pojo_TwitterResponse> GetTweets() {
+        String twitterHandle;
+        String name;
+        String message = null;
+        String profileImageUrl = null;
+        Date createdAt = null;
```
[the remark this answers is not in the document]
Comment (SupriyaPamulapati) [reply src/main/java/UTwitter/service/TwitterImplement.java]: replaced all the above inside the for loop , on considering the requirement of scope

File: src/main/models/model/Pojo_TwitterResponse.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    public String getCreated() {
+        return createdAt;
+    }
+
+    public String setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+        return createdAt;
+    }
```
[the remark this answers is not in the document]
Comment (SupriyaPamulapati) [reply src/main/models/model/Pojo_TwitterResponse.java]: sorry, i need these setters and getters to fetch data

**Pull request merged**